tools/clang/test/HLSLFileCheck/hlsl/objects/Texture/no_lod_clmap.hlsl

```diff
@@ -0,0 +1,93 @@
+// RUN: %dxilver 1.8 | %dxc -E test_sample -T ps_6_8 %s | FileCheck %s
+// RUN: %dxilver 1.8 | %dxc -E test_sampleb -T ps_6_8 %s | FileCheck %s
+// RUN: %dxilver 1.8 | %dxc -E test_sampleg -T ps_6_8 %s | FileCheck %s
+// RUN: %dxilver 1.8 | %dxc -E test_samplec -T ps_6_8 %s | FileCheck %s
+// RUN: %dxilver 1.8 | %dxc -E test_samplecb -T ps_6_8 %s | FileCheck %s
+// RUN: %dxilver 1.8 | %dxc -E test_samplecg -T ps_6_8 %s | FileCheck %s
+// RUN: %dxilver 1.8 | %dxc -E test_sample_zero -T ps_6_8 %s | FileCheck %s
+// RUN: %dxilver 1.8 | %dxc -E test_sampleb_zero -T ps_6_8 %s | FileCheck %s
+// RUN: %dxilver 1.8 | %dxc -E test_sampleg_zero -T ps_6_8 %s | FileCheck %s
+// RUN: %dxilver 1.8 | %dxc -E test_samplec_zero -T ps_6_8 %s | FileCheck %s
+// RUN: %dxilver 1.8 | %dxc -E test_samplecb_zero -T ps_6_8 %s | FileCheck %s
+// RUN: %dxilver 1.8 | %dxc -E test_samplecg_zero -T ps_6_8 %s | FileCheck %s
+
+// CHECK-NOT: Tiled resources
+
+// CHECK:define void @[[name:[a-z_]+]]()
+
+// CHECK: !dx.entryPoints = !{![[entryPoints:[0-9]+]]}
+// CHECK: ![[entryPoints]] = !{void ()* @[[name]], !"[[name]]", !{{[0-9]+}}, !{{[0-9]+}}, null}
+
+// tag 0: ShaderFlags, 4096 = Tiled resources
+// CHECK-NOT:!{i32 0, i64 4096}
+
+Texture2D T2D;
+SamplerState S;
+
+float4 test_sample(float2 coord : TEXCOORD) : SV_Target {
+  // LOD clamp requires TiledResources feature
```

Comment:
nit: These comments don't seem correct when checking the negative case.  One general comment for the file should suffice.